src/jdk.compiler/share/classes/com/sun/tools/javac/comp/Attr.java

```diff
@@ -1908,6 +1920,13 @@ private Symbol enumConstant(JCTree tree, Type enumType) {
         }
         return null;
     }
+    // where
+    static final Set<TypeTag> nonIntegralPrimitiveTypes = Set.of(
```

Comment:
This feels like a method on TypeTag?

Reply:
Addressed in https://github.com/openjdk/jdk/pull/15638/commits/9b7fb7fcdd2b393425267313622970ba709c645e